### PR TITLE
add plan dump option

### PR DESCRIPTION
**Pull request by Daniel-ND**

### CODE REVIEW

File: QueryEngine/RelAlgDagBuilder.cpp [hunk 1]
```diff
@@ -2652,6 +2653,15 @@ RelAlgDagBuilder::RelAlgDagBuilder(const std::string& query_ra,
   rapidjson::Document query_ast;
   query_ast.Parse(query_ra.c_str());
   VLOG(2) << "Parsing query RA JSON: " << query_ra;
+
+  //print plan to file
+  if (! query_plan_output_file.empty()) {
```
Comment (kurapov-peter) [inline QueryEngine/RelAlgDagBuilder.cpp:2658]: Looks like this is not formatted properly

**Review by kurapov-peter — CHANGES_REQUESTED** [review]
Please run clang-format, otherwise LGTM

File: QueryEngine/RelAlgDagBuilder.cpp [hunk 2]
```diff
@@ -35,6 +35,7 @@
 
 extern bool g_cluster;
 extern bool g_enable_union;
+std::string query_plan_output_file = {""};
```
Comment (ienkovich) [inline QueryEngine/RelAlgDagBuilder.cpp:38]: All (most of) globals used for options are started with `g_` prefix and defined in `Execute.cpp`. Can we keep it so for more unified code?

Comment (Daniel-ND) [reply QueryEngine/RelAlgDagBuilder.cpp]: fixed

**Pull request closed without merging**